src/runtime_src/core/pcie/linux/device_linux.cpp

```diff
@@ -128,6 +129,120 @@ struct bdf
   }
 };
 
+struct sdm_sensor_info
+{
+  using result_type = query::sdm_sensor_info::result_type;
+  using data_type = query::sdm_sensor_info::data_type;
+
+  static result_type
+  get(const xrt_core::device* device, key_type, const boost::any& reqType)
+  {
+    const auto req_type = boost::any_cast<query::sdm_sensor_info::req_type>(reqType);
+    auto pdev = get_pcidev(device);
+    std::string parent_path = pdev->get_sysfs_path("", "hwmon");
+    result_type output;
+    std::string errmsg;
+    std::string path;
+    bool found = false;
+
+    boost::filesystem::path render_dirs(parent_path);
+    if (boost::filesystem::is_directory(render_dirs)) {
+      boost::filesystem::recursive_directory_iterator end_iter;
+      for(boost::filesystem::recursive_directory_iterator iter(render_dirs); iter != end_iter; ++iter) {
+        std::string name;
+        std::string f_name = iter->path().filename().string();
+        if (boost::filesystem::is_directory(iter->path()) && boost::algorithm::starts_with(f_name, "hwmon")) {
+          path = "hwmon/" + f_name;
+          pdev->sysfs_get("", path + "/name", errmsg, name);
+          if (errmsg.empty()) {
+            if (boost::algorithm::contains(name, "xilinx_vck5000_gen4x8_xdma_base_1"))
+            {
+              found = true;
+              break;
+            }
+          }
+        }
+      }
+    }
+
+    if (!found) {
+      return output;
+    }
```

Comment:
No braces around single statement conditional expressions.   Also this is first place `output` is used, so just return result_type{};